app/views/docs/service.phtml

```diff
@@ -36,6 +36,19 @@ $markdown = new \Parsedown();
 $markdown->setSafeMode(true);
 
 $paramSDK = (!empty($sdk)) ? '?sdk='.$sdk : '';
+
```

Comment:
Are the service docs supposed to be working? It's showing a white screen for me.

![image](https://user-images.githubusercontent.com/1477010/207738322-79e428d8-942f-4e53-92d9-d0f64a26a87b.png)

Reply:
Should be working, give it another try after running the npm build mentioned [here](https://github.com/appwrite/homepage/pull/113#issuecomment-1352610286)

Reply:
I'm still getting some weird behavior when switching versions. I also see this in the logs:

```
appwrite-homepage-appwrite-1  | Warning: Array to string conversion in /usr/src/code/app/controllers/home.php on line 103
appwrite-homepage-appwrite-1  | 
appwrite-homepage-appwrite-1  | Fatal error: Cannot redeclare skipLanguage() (previously declared in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml:40) in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml on line 40
appwrite-homepage-appwrite-1  | [2022-12-15 18:17:11 *46.18]    ERROR   php_swoole_server_rshutdown() (ERRNO 503): Fatal error: Cannot redeclare skipLanguage() (previously declared in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml:40) in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml on line 40
appwrite-homepage-appwrite-1  | [2022-12-15 18:17:11 $8.0]      WARNING Server::check_worker_exit_status(): worker(pid=46, id=18) abnormal exit, status=255, signal=0
```

Reply:
I'm still getting some weird behavior when switching versions. I also see this in the logs:

```
appwrite-homepage-appwrite-1  | Warning: Array to string conversion in /usr/src/code/app/controllers/home.php on line 103
appwrite-homepage-appwrite-1  | 
appwrite-homepage-appwrite-1  | Fatal error: Cannot redeclare skipLanguage() (previously declared in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml:40) in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml on line 40
appwrite-homepage-appwrite-1  | [2022-12-15 18:17:11 *46.18]    ERROR   php_swoole_server_rshutdown() (ERRNO 503): Fatal error: Cannot redeclare skipLanguage() (previously declared in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml:40) in /usr/src/code/vendor/appwrite/docs/app/views/docs/service.phtml on line 40
appwrite-homepage-appwrite-1  | [2022-12-15 18:17:11 $8.0]      WARNING Server::check_worker_exit_status(): worker(pid=46, id=18) abnormal exit, status=255, signal=0
```

Reply:
Resolved the redeclaration, what was the weird behavior when switching versions? The white screen above?

Reply:
Yes, but looks like fixing the redeclaration solved the problem